_episodes/03-high-dim-modeling.md

```diff
@@ -4,43 +4,121 @@ teaching: 0
 exercises: 3
 questions:
 - "Are more features always better when trying to fit a model to your data?"
-- "What is one general way to avoid overfitting when training on high-dimensional data?"
+- "What does a model's level of bias or variance indicate?"
+- "What are some of the popular methods to avoid overfitting when training on high-dimensional data?"
 - "How can one determine which features are most relevant to a model's predictions?"
 objectives:
 - "Understand the challenges associated with modeling high-dimensional data"
 - "Understand the importance of feature selection as a tool for modeling high-dimensional data"
-- "Identify and understand some of the possible ways to perform feature selection"
+- "Identify and understand different approaches/methods for feature selection"
+- "Understand the limitations of feature selection techniques and ways to assess model bias/variance tradeoff"
+- "Learn how to fit and interpret univariate and multivariate linear models"
 keypoints:
 - "First key point. Brief Answer to questions. (FIXME)"
 ---
-FIXME
-
 {% include links.md %}
+# Schedule Outline (internal use; will remove this later)
+1. Intro/overview of ML
+      * What goes into a model/estimator
+      * What is the goal of the model (prediction or classification)
+      * How is a model trained? Optimizing error functions.
+      * Intro to dataset and linear modeling
+      * Live-coding: Training a univariate linear model
+2. The curse of dimensionality
+      * Overview
+          * Overfitting, bias vs. variance trade-off discussion
+          * How to assess overfitting
+          * Comment on interpretability and training time of high-dimensional models
+      * Live-coding: Use same model given in intro section but add additional predictor variables to demonstrate overfitting
+3. Methods for feature selection
+      * Intro to feature selection; exercises to illustrate the concept of feature selection
+      * Filter methods
+          * Overview
+          * Live-coding: implement varianceThreshold and selectKbest (F-test and mutual information tests) using sklearn
+      * Wrapper methods
+          * Overview of forward selection
+          * Live-coding: implement forward selection using mlxtend library (built in wrapper methods)
```

Comment:
Maybe these notes sections would be better kept in the Instructor notes while they are needed?